src/contractTest/java/uk/gov/hmcts/reform/waworkflowapi/consumer/wa/WorkflowPostMessageConsumerTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.waworkflowapi.consumer.wa;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.http.ContentType;
+import net.serenitybdd.rest.SerenityRest;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.hmcts.reform.waworkflowapi.SpringBootContractBaseTest;
+import uk.gov.hmcts.reform.waworkflowapi.clients.model.SendMessageRequest;
+import uk.gov.hmcts.reform.waworkflowapi.provider.service.CamundaConsumerApplication;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.waworkflowapi.clients.model.DmnValue.dmnStringValue;
+
+@PactTestFor(providerName = "wa_workflow_api_post_message", port = "8899")
+@ContextConfiguration(classes = {CamundaConsumerApplication.class})
+@TestPropertySource(locations = {"classpath:application.properties"})
+public class WorkflowPostMessageConsumerTest extends SpringBootContractBaseTest {
```

Comment:
Can we update this consumer name to match the name of the method on the CreateTaskController class? I.e. sendMessage

Reply:
class renamed